examples/todo-list/nextjs-todo-list/README.md

```diff
@@ -35,6 +33,73 @@ The `anon` key is your client-side API key. It allows "anonymous access" to your
 
 ## Supabase details
 
+### Local Development Stack
```

Comment:
@thorwebdev is this change ok? I assume devrel is maintaining these examples.

We are trying to update everything to have use a config.toml file for configuring services.

Reply:
In my mind the user would probably do both. Setting up things locally and have a feel of how you can develop/change them. Then try to get it deployed. But I can see how it can come across as confusing.